prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -1342,3 +1342,31 @@ periodics:
       - test-e2e-go-gke-ci
       - 'E2E_ARGS=--stress -run=TestStress*'
       - 'GCP_CLUSTER=stress-test'
+
+- name: vulnerability-scan
+  interval: 1h
+  annotations:
+    testgrid-dashboards: googleoss-kpt-config-sync-misc
+    testgrid-tab-name: vulnerability-scan
+  cluster: build-kpt-config-sync
+  decorate: true
+  decoration_config:
+    timeout: 1h
+  extra_refs:
+  - org: GoogleContainerTools
+    repo: kpt-config-sync
+    base_ref: main
```

Comment:
Just to confirm the vulnerability postsubmit job only runs against `main`?

Reply:
Yes this job currently only runs on `main`, but the intention is to add another job definition for the release branch once:
- the postsubmit changes are on the release branch (e.g. cherrypick) and the postsubmit job is running on the release branch
- the vulnerability scanning changes are on the release branch (e.g. cherrypick)

Reply:
I'm wondering whether we need a job for the release branch. The release branch is created based off the main branch, and all commits are cherry-picked from main. If the main branch has no vulnerabilities, the release branch should have none, too. If the main branch has vulnerabilities, the release branch may or may not have, but eventually, the vulns should be fixed on main for the next minor release.

Reply:
I think we should still have a job for the release branch, as even if we are cherry picking changes to the release branch it's not guaranteed that all artifacts/dependencies will be identical between refs. We should verify artifacts from the exact commit SHA to be certain

Reply:
Make sense